.github/workflows/codeql.yml

```diff
@@ -48,8 +48,8 @@ jobs:
 
       - name: Install project
         run: |
-          pip install --editable=.[sqlalchemy,test,doc]
-          pip install "sqlalchemy==${{ matrix.sqla-version }}" --upgrade --pre
+          pip install --editable=.[sqlalchemy,test]
```

Comment:
just double checking, is removing `doc` correct?

Reply:
Thanks for asking. Yes, the `doc` extra had to be removed because its dependencies depressed `urllib3` to `<2`, rendering GH-510 useless.

The dependencies are not required on CI/GHA for those jobs, there is a separate workflow definition file [`docs.yml`](https://github.com/crate/crate-python/blob/master/.github/workflows/docs.yml) for that.